### DISCUSSION TITLE
Esiste una versione "portable" di Visidata?

### DISCUSSION THREAD

**Discussion by lorenzoperone** [ondata/guidaVisiData#38] [Domande]
Nel mio portatile di lavoro, che uso abitualmente nell'analisi dati non ho i privilegi per installare applicazioni (Python, pip) esiste una versione "portable" di Visidata?

**Comment by aborruso** [comment]
Caro @lorenzoperone al momento no.

Ma non esiste un python portable, installabile nella tua cartella utente e soltanto su quello provare a installare?

Ci sono delle modalità per compilare progetti python e creare un eseguibile (io ho usato [questo](https://www.pyinstaller.org/) per una cosa piccola, che uso ogni giorno). Ma potrebbe essere complesso per un progetto così ricco.

Prova a fare la proposta [qui](https://github.com/saulpw/visidata/discussions), spiegando le tue esigenze comuni a molti tipi di utenti aziendali.

Ciao

**Comment by saulpw** [comment]
Hello!  I was eavesdropping on this conversation.  I think having a VisiData.exe that people could run/install on Windows would be great.  I actually built one this past year--but no one else could run it, because Windows requires all binaries to be signed (which is a hassle and costs money every year for the certificate).  I tried going through the Windows Store to make some of this a bit easier, but that apparently wouldn't even remove the big security warning dialog, until enough people have downloaded and run the binary successfully--and this would reset every release.

There are other smaller issues too with certain features and use cases, and as I'm not a Windows developer or user, I would not be able to support it and give it the required attention.  If someone was willing to be the Windows Release Manager for VisiData, we would certainly support them in every way!  But I'm afraid we don't have the skills or time to create a proper Windows release ourselves.

**Comment by lorenzoperone** [comment]
Hi,
thanks to Andrea and Saul.
I've installed [miniconda](https://docs.conda.io/en/latest/miniconda.html) without admin privileges and then I've installed Visidata as described [here](https://libraries.io/conda/visidata).
But I can't find the way tu run it :)
Lorenzo

**Reply by aborruso** [reply]
You should open the shell (probably there is a miniconda shell, with all path variables mapped) and run `vd`.

**Reply by aborruso** [reply]
@lorenzoperone did you do any tests? How did it go?

**Comment by lorenzoperone** [comment]
Hi,
I've done a couple of tests without any positive results :(
I've tried to launch the program via Conda and Anaconda.
L.
